src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -0,0 +1,286 @@
+
+
+### TLS 1.3
+
+TLS 1.3 is the newest version of [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security), the cryptographic protocol used by [HTTPS](https://en.wikipedia.org/wiki/HTTPS). TLS 1.3 is more secure than TLS 1.2 and also shortens the TLS handshake from two roundtrips to one. For connections using HTTP/1 or HTTP/2, shortening the TLS handshake to one roundtrip effectively reduces connection setup time by 33%.
+
+<figure class="w-figure">
+  <img src="./cdn3.png" alt="Comparison of the TLS 1.2 and TLS 1.3 handshakes" class="w-screenshot">
+</figure>
+
+### HTTP/2 & HTTP/3
+
+HTTP/2 and HTTP/3 both provide performance benefits over HTTP/1. Of the two, HTTP/3 offers greater _potential_ performance benefits. HTTP/3 isn't fully standardized yet, but it will be widely [supported](https://caniuse.com/#feat=http3) once this occurs.
+
+
+#### HTTP/2
+
+If your CDN hasn't already enabled [HTTP/2](https://en.wikipedia.org/wiki/HTTP/2) by default, you should consider turning it on. HTTP/2 provides multiple [performance benefits](https://hpbn.co/http2) over HTTP/1 and is [supported](https://caniuse.com/#feat=http2) by all major browsers. Performance features of HTTP/2 include: [multiplexing](https://hpbn.co/http2/#request-and-response-multiplexing), [stream prioritization](https://hpbn.co/http2/#stream-prioritization), [server push](https://developers.google.com/web/fundamentals/performance/http2), and [header compression](https://blog.cloudflare.com/hpack-the-silent-killer-feature-of-http-2/).
+
+
+*  **Multiplexing**
+
+    Multiplexing is arguably the most important feature of HTTP/2. Multiplexing enables a single TCP connection to serve multiple request-response pairs at the same time. This eliminates the overhead of unnecessary connection setups; given that the number of connections that a browser can have open at a given time is limited, this also has the implication that the browser is now able to request more of a page's resources in parallel. Multiplexing theoretically removes the need for HTTP/1 optimizations like concatenation and sprite sheets - however, in practice, these techniques will remain relevant given that larger files compress better.
+
+
+*  **Stream Prioritization**
+
+    Multiplexing enables multiple concurrent streams; [stream prioritization](https://http2.github.io/http2-spec/#StreamPriority) provides an interface for communicating relative priority of each of these streams. This helps a server decide how to allocate its limited resources. Stream prioritization is expressed by the browser via a dependency tree and is merely a statement of _preference_: in other words, the server is not obligated to meet (or even consider) the priorities supplied by the browser.
```

Comment:
The sequencing of how resources are loaded has a significant impact on how long it takes for a user to see and interact with a page. The ability to use prioritization allows the sever to send the most important resource before less important resources even if the important resource was requested later such as a font. The more of the site that is served through a CDN the more effective the prioritization becomes.